exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function first
+function numbers(a, b) {
```

Comment:
Small nitpick, but "numbers" is not a very descriptive function name, "add" might be better. (You'd be surprised how non-trival amount of coding is coming up with good names...)